src/main/java/com/yahoo/identity/Identity.java

```diff
@@ -14,21 +14,20 @@
 public class Identity {
 
     private final AccountService accountService;
+    private final SessionService sessionService;
 
-    public Identity(@Nonnull AccountService accountService) {
+    public Identity(@Nonnull AccountService accountService, @Nonnull SessionService sessionService) {
         this.accountService = accountService;
+        this.sessionService = sessionService;
     }
 
     @Nonnull
     public AccountService getAccountService() {
-        return accountService;
+        return this.accountService;
     }
 
     @Nonnull
-    public SessionService getSessionService() {
-        //TODO
-        return null;
-    }
+    public SessionService getSessionService() { return sessionService; }
```

Comment:
Don't put in single line.

Reply:
Already reformatted based on the code style xml.